tensorflow/stream_executor/cuda/cuda_diagnostics.cc

```diff
@@ -315,7 +315,8 @@ port::StatusOr<DriverVersion> Diagnostician::FindKernelDriverVersion() {
     // NOTE: OSX CUDA driver does not currently store the same driver version
     // in kCFBundleVersionKey as is returned by cuDriverGetVersion
     CFRelease(kext_infos);
-    const CFStringRef str = (CFStringRef)CFDictionaryGetValue(cuda_driver_info, kCFBundleVersionKey);
+    const CFStringRef str =
+      (CFStringRef)CFDictionaryGetValue(cuda_driver_info, kCFBundleVersionKey);
```

Comment:
Oh cool, forgot about clang-format. PS, this file is quite un-clang-formatted otherwise